scripts/rook.sh

```diff
@@ -68,6 +69,7 @@ function deploy_rook() {
         kubectl_retry create -f "${ROOK_URL}/toolbox.yaml"
         kubectl_retry create -f "${ROOK_URL}/filesystem-test.yaml"
         kubectl_retry create -f "${ROOK_URL}/pool-test.yaml"
+	kubectl_retry create -f "${ROOK_URL}/pool-ec.yaml"
```

Comment:
ah, here it is always created... and you want that to be only done when calling `rook.sh create-ec-pool` (note the `-` instead of `_` too!)

Reply:
Updated it now. Thanks!